mlflow/entities/model_registry/prompt_info.py

```diff
@@ -0,0 +1,71 @@
+"""
+The prompt info entity for Unity Catalog MLflow Model Registry.
+"""
+
+from typing import Dict, List, Optional
+
+
+class PromptInfo:
```

Comment:
Can we place this in some other undocumented module, e.g. in `mlflow/store/_unity_catalog/registry/` where it's used?

Otherwise, this becomes a public, documented, user-facing entity. I don't think we're ready to commit to that yet.